libs/utils/website/src/lib/inject-script.tsx

```diff
@@ -0,0 +1,45 @@
+import { ApiV1 } from '@wepublish/website'
+import {useEffect} from 'react'
+
+export function InjectScript() {
+  const {data} = ApiV1.useSettingQuery({
+    variables: {
+      name: 'bodyScript'
+    }
+  })
+
+  /* Because of the WhatWG HTML specs, article 4.12.1: "When inserted using the
+  innerHTML and outerHTML attributes, they do not execute at all.", the scripts
+  must be injected differently. The below is one such method and works by
+  copying the entire element (tagname, attributes and content) and injecting
+  the copy using appendChild.
+  */
+  useEffect(() => {
```

Comment:
Can not be server rendered now

Maybe a bit of a crazy idea but could we store a JSON in the database like:

```
[
  {
     "src": "https://example.com/example.js",
     "data-id": "1234",
     "async": true
   }
]
```

But display the HTML in the editor (and parse from pasted HTML)? This would allow the frontend to take the JSON and properly render the script tags on the server (use the `Script` component in the website builder).

I think this is the cleanest in terms of structured data, the frontend then also doesn't rely on browser APIs (like createElement) so we can run it on the server too.